pinakes/settings/defaults.py

```diff
@@ -33,7 +33,9 @@
 DEBUG = env.bool("PINAKES_DEBUG", default=False)
 TEMPLATE_DEBUG = DEBUG
 SQL_DEBUG = DEBUG
+
 SECRET_KEY = env.str("PINAKES_SECRET_KEY")
+DB_ENCRYPTION_KEYS = env.str("PINAKES_DB_ENCRYPTION_KEYS").split(":")
```

Comment:
I don't think so. The secret key setting `SECRET_KEY` is specifically used by Django for cryptographic signing (e.g. sessions, caches, password reset tokens):

> A secret key for a particular Django installation. This is used to provide [cryptographic signing](https://docs.djangoproject.com/en/4.0/topics/signing/), and should be set to a unique, unpredictable value.

https://docs.djangoproject.com/en/4.0/ref/settings/#secret-key

While technically it can be re-used as an encryption key as well, there are several pitfalls that makes its usage more complicated.

1. `SECRET_KEY` doesn't match the Fernet key requirements. Fernet expects a 256-bit base64-encoded key, while `SECRET_KEY` can be an arbitrary string. It can be converted into an appropriate Fernet key though by using key derivation function (KDF).
2. `SECRET_KEY` value is disposable. A site administrator can safely rotate `SECRET_KEY` value without data loss, since it's used primarily for signing temporary data (e.g. sessions, caches, password reset tokens). Using this setting for encryption requires defining additional restrictions, which may break site administrator expectations.
3. Encryption key rotation process requires extra precautions. Old key cannot be disposed before all data is re-encrypted.
4. `SECRET_KEY` stores only single value. To perform zero downtime key rotation process multiple keys (a new one and existing one) must be available.